.github/partial-readmes/SecurityCodeComponent.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Do you want this file empty?

Reply:
no, using Github's UI to create it now 🙂